src/pages/deck/FlashcardForm.js

```diff
@@ -0,0 +1,113 @@
+import React, { useState } from "react";
+import "./Deck.css";
+
+const CARDS = [
+  {
+    id: 1,
+    question: "test Question1",
+    answer: "test Answer",
+    hint: "Hint",
+  },
+  {
+    id: 2,
+    question: "test Question2",
+    answer: "test Answer",
+    hint: "Hint",
+  },
+  {
+    id: 3,
+    question: "test Question 3",
+    answer: "test Answer",
+    hint: "Hint",
+  },
+  {
+    id: 4,
+    question: "test Question 4",
+    answer: "test Answer",
+    hint: "Hint",
+  },
+];
+
+const FLASHCARD_FIELDS = {
+  question: "",
+  answer: "",
+  hint: "",
+  id: 0,
+};
+function FlashCardForm() {
+  let [flashCard, setFlashCard] = useState(FLASHCARD_FIELDS);
+  let [cards, setCards] = useState(CARDS);
+
+  const handleSaveCard = () => {
+    //TODO: save card on backend
+    let updated = [...cards, flashCard];
+    setCards(updated);
+
+    //clear input after savings
+    setFlashCard(FLASHCARD_FIELDS);
+  };
+
+  const handleDelete = (index) => {
```

Comment:
Add a call to the delete flashcard route. Look in the project tracker for the route URL - it needs to include the _id of the flashcard you want to delete.

This _id is sent back in the response object when a flashcard is created.